src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
```

Comment:
`@ignore` will prevent edoc (or what's used for generating documentation for riak) from looking at this comment as pertaining to the function that follows. `DELETE` (in capitals) is to be understood in SQL-ish sense (there are `CREATE TABLE` and `SELECT` at other heads of `process/2`. And, `as if` refers to the fact that these are separate API calls, not string arguments to `query/2`.

Eventually, I removed all those cryptic comments, because the code is self-explanatory as to what messages `process` handles (from the #tsdothing arg).